services/static-badge/query-string-static.service.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+const { BaseStaticService, InvalidParameter } = require('..')
+
+module.exports = class QueryStringStaticBadge extends BaseStaticService {
+  static get category() {
+    return 'other'
+  }
+
+  static get route() {
+    return {
+      format: 'static/v([0-9])',
+      capture: ['schemaVersion'],
+      // All but one of the parameters are parsed via coalesceBadge. This
+      // reuses what is the override behaviour for other badges.
+      queryParams: ['message'],
+    }
+  }
+
+  handle(namedParams, queryParams) {
```

Comment:
It would be good to validate the `queryParams` using Joi, as we've done in e.g. the dynamic badge. It will ensure that `message` is a string or numeric string, and not e.g. an array. (same issue as #2854)

Reply:
I added validation as best as I could figure out how. I'm sure there's a better way.